exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -225,9 +225,9 @@ private ExecConstants() {
   public static final String HTTP_SPNEGO_PRINCIPAL = "drill.exec.http.auth.spnego.principal";
   public static final String HTTP_SPNEGO_KEYTAB = "drill.exec.http.auth.spnego.keytab";
   //Control Web UI Resultset
-  public static final String HTTP_WEB_CLIENT_RESULTSET_AUTOLIMIT_ENABLE = "drill.exec.http.web.client.resultset.autolimit.enable";
+  public static final String HTTP_WEB_CLIENT_RESULTSET_AUTOLIMIT_CHECKED = "drill.exec.http.web.client.resultset.autolimit.checked";
   public static final String HTTP_WEB_CLIENT_RESULTSET_AUTOLIMIT_ROWS = "drill.exec.http.web.client.resultset.autolimit.rows";
-  public static final String HTTP_WEB_CLIENT_RESULTSET_DEFAULTROWSPERPAGE = "drill.exec.http.web.client.resultset.defaultRowsPerPage";
+  public static final String HTTP_WEB_CLIENT_RESULTSET_ROWS_PER_PAGE = "drill.exec.http.web.client.resultset.rowsPerPage";
```

Comment:
This parameter now represents all the options for rows per page not a single value so consider naming it `rowsPerPageOptions` / `rowsPerPageValues`

Reply:
+1 Will change it to `rowsPerPageValues`